swarm/tools/flow_studio_ui/fragments/60-modals.html

```
@@ -70,5 +70,7 @@ <h3 id="shortcuts-modal-title">Keyboard Shortcuts</h3>
     <div id="run-detail-modal-content">
       <div class="muted">Loading...</div>
     </div>
+    <!-- Hidden rerun button for test automation -->
+    <button id="run-detail-rerun-btn-hidden" data-uiid="flow_studio.modal.run_detail.rerun" style="display: none;">Re-run</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)